notify/api/v1/api.py

```diff
@@ -15,21 +15,44 @@
 
 import logging
 
+import jsonschema
 import flask
 
 from notify import config
-
+from notify.drivers.salesforce import salesforce
 
 LOG = logging.getLogger("api")
 LOG.setLevel(config.get_config().get("logging", {}).get("level", "INFO"))
 
+notify_backends_config = config.get_config()['notify_backends']
+sfdc_config = notify_backends_config['sf']['salesforce']['properties']
```

Comment:
This is wrong, there is no rules that "sf" has "salesforce" type of driver (it may have)